resources/migrations/0108_auto_20230711_1036.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.11 on 2023-07-11 07:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('resources', '0107_auto_20230609_0834'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='resource',
+            name='need_manual_confirmation_for_zero_price',
+            field=models.BooleanField(default=False, verbose_name='Need manual confirmation when price is zero'),
+        ),
+        migrations.AlterField(
```

Comment:
Looks like these changes are not from this PR 🤔 I guess there were some model changes without matching migrations.

Reply:
The `--name` flag is generally good practice, I agree, although one that hasn't been followed consistently in this project. However as you pointed out there was also a pending auto migration of a field (Django annoyingly does this even for a non-DB modification such as a verbose name  or help text change) so the name wouldn't be entirely accurate. We could maybe avoid this by splitting out into multiple migrations, although that would make migrations in deployment a bit slower.